cdap-ui/app/cdap/components/AbstractWidget/HierarchyWidget/__tests__/HierarchyWidget.tests.ts

```diff
@@ -0,0 +1,419 @@
+/*Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { typeHandler, flatToTree, addPathToFlatMap, toJsonHandler } from '../dataHandler';
+
+describe('Unit tests for Hierarchy Widget', () => {
```

Comment:
nit: This is unit test for data handler helper function in Hierarchy widget right? We are not necessarily testing the hierarchy widget.